### DISCUSSION TITLE
Strawberry Shake: How to have Optional Parameters in Queries

### DISCUSSION THREAD

**Discussion by Eschryn** [ChilliCream/hotchocolate#5864] [Q&A]
When you pass null it is treated as null value and gets serialized - how can I have optional parameters (not getting serialized in the Query Params -> undefined for js) in Strawberry Shake 13?

**Comment by dillon-miller2-carvana** [comment]
Any updates on this? This is causing issues when attempting to calling a GraphQL API from within another GraphQL API (without the use of stitching).
